### ISSUE TITLE
Finetuning on Downstream Tasks: Eval zero-shot perplexities on standard evals (e.g. LAMBADA? HELM? etc.)

### ISSUE THREAD

**Issue by BiEchi** [karpathy/nanoGPT#385]
Excuse me, just want to ask whether there's any progress on Eval zero-shot perplexities on standard evals (e.g. LAMBADA? HELM? etc.). We're using this repo for downstream eval to show the model works. Looking forward to your reply!

**Comment by VatsaDev** [comment]
I don't GPT-2 was really meant for benchmarks other than perplexities on datasets? You could get the train/val loss when finetuning on a benchmarks train split, while using val split for val loss, or building a custom eval script, as I dont think this works with lm-harness?
